odoo/addons/base/tests/test_db_cursor.py

```diff
@@ -31,6 +33,15 @@ def test_execute_bad_params(self):
             with self.assertRaises(ValueError):
                 cr.execute("SELECT id FROM res_users WHERE id=%s", '1')
 
+    @mute_logger('odoo.sql_db')
+    def test_normal_cursor(self):
+        with registry().cursor() as cr:
+            cr.close()
+            with self.assertRaises(psycopg2.InterfaceError):
```

Comment:
Isn't there a more precise error?

Reply:
No, it is the same error raised by psycopg2 for its closer cursors.